config.md

```diff
@@ -118,6 +118,10 @@ For more information about SELinux, see  [Selinux documentation](http://selinuxp
 * **`noNewPrivileges`** (bool, optional) setting `noNewPrivileges` to true prevents the processes in the container from gaining additional privileges.
 [The kernel doc](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt) has more information on how this is achieved using a prctl system call.
 
+For Windows-based systems, the process structure supports the following process specific fields:
```

Comment:
In linux, we also use a default winsize for the tty console when starting the container, and then we resize it later, these is a gap between them.